### DISCUSSION TITLE
Remove ".html" extension

### DISCUSSION THREAD

**Discussion by bridevmx** [pocketbase/pocketbase#6082] [Q&A] [answered]
I am serving html pages from "pb_public", my question is if it is possible to omit the ".html" extension from the routes, for example /home.html -> /home

**Comment by ganigeorgiev** [comment, marked as answer]
We don't have a builtin rewrite rule for this, if that is what you are asking, but you can:

- use a reverse proxy (e.g. nginx, apache, caddy)
- define a custom route that will accept a path without html and will automatically append the suffix before retrieving the file
- OR rename your `home.html` to `index.html` and place it inside `home` directory (SvelteKit for example has the `trailingSlash` option that do this automatically - https://svelte.dev/docs/kit/page-options#trailingSlash)
